lib/parse.js

```diff
@@ -84,6 +135,35 @@ var INLINE_CODE = 'inlineCode';
 var BREAK = 'break';
 var ROOT = 'root';
 
+/** TODO */
+function isWhiteSpace(character) {
```

Comment:
I can imagine, thanks for worrying haha! I think you’re right, we’ll see. I’ll have to see where things go from here. I might just revert that change, but we’ll have to see where things go when everything is transformed to the new “algorithmic” approach!

First I’m going overboard with this, then we’ll see which cases can return to `\s` like regexes! :smile: